fiftyone/utils/muse_glimmer.py

```
@@ -0,0 +1,386 @@
+
+        if kwargs.get("device_map") is None and self._using_gpu:
+            model = model.to(self._device)
+        model.eval()
+
+        self._processor = transformers.AutoProcessor.from_pretrained(
+            config.name_or_path
+        )
+
+        return model
+
+    def _get_prompt(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_get_prompt`

(ANN202)